src/App.tsx

```diff
@@ -392,6 +400,7 @@ export class App extends React.Component<RouteComponentProps, State> {
       this.setStoredState()
       this.props.history.push(path)
     })
+    this.startPolling()
```

Comment:
It doesn't need to be as there is no need to wait until the `userState` is updated before starting to poll again. The callback to setState only ensures that the callback happens after the state is updated. The polling will trigger `proccessCardData` which will write over the necessary state. 

Because the callback it unrelated to the polling, lets keep them separate.

Reply:
cool, 👍